elastic/kibana
# [Security Solution][Detection Engine] deprecates siem signals migration APIs

## Summary

 - addresses part of https://github.com/elastic/kibana/issues/195523
 - deprecates https://www.elastic.co/guide/en/security/current/signals-migration-api.html APIs according to internal 9.x readiness [guideline](https://docs.google.com/document/d/1W7csjn6QYjrBjmbXMzSz_JUD4KcmWz8jTTtAWFwgUJM/edit?tab=t.0#heading=h.tui2zvb9gca6)
 
#### How to test deprecated APIs?

1. Run API https://www.elastic.co/guide/en/security/current/signals-migration-api.html
2. Observe warning deprecation on Kibana Upgrade page

<img width="2540" alt="Screenshot 2024-12-03 at 10 43 59" src="https://github.com/user-attachments/assets/24fcebb9-2d31-4ca3-a0dc-4ed7861d26a2">
